mercury/templates/index.html

```diff
@@ -1,46 +1,60 @@
 {% load static %}
 <link rel="stylesheet" type="text/css" href="{% static 'mercury/style.css' %}">
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
```

Comment:
Is this being used anywhere?

Reply:
I don't believe that is being used anywhere-- that was an older code that would've been used in the carousel.